jujugui/static/gui/src/app/components/entity-content/entity-content.js

```diff
@@ -150,7 +150,7 @@ YUI.add('entity-content', function() {
         var submitLink = 'https://bugs.launchpad.net/charms/+source/' +
           `${entityModel.get('name')}/+filebug`;
         var contributeLink = 'https://code.launchpad.net/~charmers/charms/' +
-          `trusty/${entityModel.get('name')}/trunk`;
+          `${entityModel.get('series')}/${entityModel.get('name')}/trunk`;
```

Comment:
TL;DR: All of these links are actually wrong and will get even further from the truth as we carry on.

You can add a 'homepage' and 'bug-url' to your charms by running:
`charm set juju-gui --channel development homepage=https://github.com/juju/juju-gui/`
The information is then available at:
`https://api.jujucharms.com/charmstore/v4/juju-gui/meta/common-info`

I believe there is only a single charm right now with this information so we need to open this up to more discussion. I'd say that this issue is out of scope for this PR because technically all of these links "bugLink, submitLink, and contributeLink" will be wrong.